account.py

```diff
@@ -0,0 +1,47 @@
+class BankAccount(object):
+    def __init__(self, account_number, name, pin_number):
+        self.status = "open"
+        self.account_number = account_number
+        self.name = name
+        self.pin_number = pin_number
+    
+    def __repr__(self):
+        return "".format(BancAccount (self.account_number, self.name))
```

Comment:
This is a 🐛 (BUG) `BancAccount` does exist anywhere. If you were to use this code `.format` it would be better if you used it like this
```py
 # the two {}  act as placeholders for the name and account numbers
"<BankAccount {} {}>".formart(self.name, self.account_number)
```

Reply:
Read more about using `.formart` here https://pyformat.info/